files/zh-cn/web/html/element/meta/name/index.md

```diff
@@ -95,7 +95,7 @@ The [WHATWG Wiki MetaExtensions page](https://wiki.whatwg.org/wiki/MetaExtension
 - `publisher`：当前文档的发布者/出版者。
 - `robots`：爬虫、协作搜寻器，或者“机器人”，对此页面的处理行为，或者说，应当遵守的规则。是一个使用逗号分隔的、由下列值构成的列表：
 
-  | 值             | 描述                                                                        | 被用于                                                                                                                                                                                                                                                                                           |
+  | 值             | 描述                                                                        | 适用于                                                                                                                                                                                                                                                                                           |
```

Comment:
The previous translation is precise. As bots are [_using those metadata_](https://developers.google.com/search/docs/crawling-indexing/robots-meta-tag#robotsmeta).

We also use ["used by" in en-US](https://github.com/mdn/content/blob/e125a8325caadaa3799a1c9b484a728aa8698aff/files/en-us/web/html/element/meta/name/index.md?plain=1#L227).

Reply:
got it.